### ISSUE TITLE
Implement Java algorithms or data structures!

### ISSUE THREAD

**Issue by AjinkyaTaranekar** [AjinkyaTaranekar/AlgorithmX#7]
Hello everyone Hacktoberfest 2020 is going on ! Let's get started with JAVA, implement algorithms, data structures, or even post SDE interview questions too!

But remember, not to spam with PRs with simple white-space removing and all.

**Comment by arpita2712** [comment]
May I create a PR for Bubble Sort implementation in Java ?

**Comment by NirmalSilwal** [comment]
Hi @AjinkyaTaranekar, this is great idea. I am not sure whether adding algorithms / data structures / coding problems will be counted towards valid PR?

Also there has been [update on efforts to reduce spam with Hacktoberfest](https://hacktoberfest.digitalocean.com/hacktoberfest-update) , hope you have gone through it. This repo need to add **hacktoberfest** label to consider for hacktoberfest.

What are your suggestions?

**Comment by AjinkyaTaranekar** [comment]
@NirmalSilwal Yes it will be, but just copying from GFG wont be enough, try to maintain originality. Yes I was reading those updates only. Thanks for suggestions.

**Comment by NirmalSilwal** [comment]
Yes, I am looking to add my approach to coding problem. Will that be fine?

**Comment by AjinkyaTaranekar** [comment]
Yeah Sure!

**Comment by bhavyakjain** [comment]
I am interested in contributing my solutions for LeetCode/HackerRank/CodeChef problems.

**Comment by debarshi-1999** [comment]
I want to implement Quick sort using JAVA. Please assign me this issue.

**Comment by icancodealright** [comment]
I am new to Github so i have few Question like.
1.I want to implement Generic Linked List so i want to know how this works like ,I can fork the repo and make the pull request or 
   @AjinkyaTaranekar you will assign the algorithm. 
  Thank you 
  Happy Hacking
